src/lib/vocab/account/accountEvents.ts

```diff
@@ -0,0 +1,26 @@
+import type {ServiceEventInfo} from '$lib/vocab/event/event';
+
+export const UpdateAccountSettings: ServiceEventInfo = {
+	type: 'ServiceEvent',
+	name: 'UpdateAccountSettings',
+	broadcast: true,
```

Comment:
```suggestion
	broadcast: false,
```

I think we'll want to broadcast this to all connected clients for this account, and that may reuse `broadcast: true` with proper scoping (scopes I can think of: account, community, space), but for now it'll be less of a security problem to just make it not broadcast.